armory/eval/evaluator.py

```diff
@@ -135,7 +135,7 @@ def _cleanup(self):
             else:
                 json = ""
             output_path = os.path.join(self.output_dir, json)
-            log.info(f"results output written to:\n{output_path}")
```

Comment:
The '\n' was added to make it easier to grab the path of the outputs after running. Revert